crates/services/tx_status_manager/src/service.rs

```diff
@@ -106,6 +116,12 @@ impl SharedData {
         let request = WriteRequest::NotifySkipped { tx_ids_and_reason };
         let _ = self.write_requests_sender.send(request);
     }
+
+    pub fn subscribe_all_updates(
+        &self,
+    ) -> tokio::sync::broadcast::Receiver<(TxId, TransactionStatus)> {
```

Comment:
You could use `broadcast::Receiver` instead of the full full path=)

Reply:
Done